tests/test_examples.py

```diff
@@ -524,6 +525,32 @@ def test_register_envs(tmp_path, request):
         assert str(install_dir) not in environments_txt
 
 
+@pytest.mark.skipif(sys.platform != 'darwin', reason='MacOS only')
+@pytest.mark.parametrize('domains', ({}, {'enable_anywhere': 'false', 'enable_localSystem': True}))
+def test_pkg_distribution_domains(tmp_path, domains):
```

Comment:
Do you think we should still try to install the PKG in this test to make sure `pkg_domains` work?

Reply:
The domains only affect the GUI installer, not the CLI. You can select `CurrentUserHomeDirectory` and `LocalSystem` as the target, no matter what `Distributions` says.